src/Scripts/model/i18n/nl.ts

```diff
@@ -333,19 +333,19 @@ const locale: Locale = {
         [_.saveVpaxCtrlTile]: "Opslaan als VPAX",
         [_.savingVpax]: "Genereer VPAX...",
         [_.sceneUnsupportedReason]: "Deze functionaliteit is niet beschikbaar voor deze data bron.",
-        [_.sceneUnsupportedReasonManageDatesAutoDateTimeEnabled]: `Modellen met automatische datum/tijd-optie worden niet ondersteund.<br><span class="link" href="https://www.sqlbi.com/tv/disabling-auto-date-time-in-power-bi/">Automatische datum-tijd uitschakelen in Power BI (video)</span>`,
+        [_.sceneUnsupportedReasonManageDatesAutoDateTimeEnabled]: "Modellen met automatische datum/tijd-optie worden niet ondersteund.<br><span class="link" href="https://www.sqlbi.com/tv/disabling-auto-date-time-in-power-bi/">Automatische datum-tijd uitschakelen in Power BI (video)</span>`,
```

Comment:
Can you change the first character to be double quote instead of single quote ? 
Should be `` `Modellen met `` instead of `` "Modellen met ``